sentry/const.py

```diff
@@ -65,19 +67,31 @@ def select_transport(name=DEFAULT_TRANSPORT):
     }.get(name, HttpTransport)
 
 
-def get_sentry_logging(level=DEFAULT_LOG_LEVEL):
-    if level not in LOG_LEVEL_MAP:
-        level = DEFAULT_LOG_LEVEL
+def get_log_level(log_level, default):
+    if log_level not in LOG_LEVEL_MAP:
+        log_level = default
+    return LOG_LEVEL_MAP[log_level]
+
 
-    return LoggingIntegration(level=LOG_LEVEL_MAP[level], event_level=logging.WARNING)
+def get_sentry_logging(config):
+    return LoggingIntegration(
+        level=get_log_level(
+            config.get("sentry_breadcrum_logging_level"), DEFAULT_BREADCRUM_LOG_LEVEL
+        ),
+        event_level=get_log_level(
+            config.get("sentry_event_loggin_level"), DEFAULT_EVENT_LOG_LEVEL
+        ),
+    )
 
 
 def get_sentry_options():
     return [
         SentryOption("dsn", "", str.strip),
         SentryOption("transport", DEFAULT_OPTIONS["transport"], select_transport),
-        SentryOption("logging_level", DEFAULT_LOG_LEVEL, get_sentry_logging),
         SentryOption("with_locals", DEFAULT_OPTIONS["with_locals"], None),
+        SentryOption(
+            "include_local_variables", DEFAULT_OPTIONS["include_local_variables"], None
```

Comment:
I got a `KeyError` here. Do you use a different sentry-sdk version? The manifest still says `sentry_sdk<=1.9.0`

Reply:
Yes we use newer sentry_sdk in v14.

I dislike setting an hard limit on manifest like that. Newer version of sentry can run perectly fine. BUT you may need to update other libs as well.

We use https://github.com/camptocamp/docker-odoo-project/blob/master/14.0/base_requirements.txt